python/ray/scripts/scripts.py

```diff
@@ -1123,13 +1123,14 @@ def on_terminate(proc):
     grace_period_to_kill_gcs = int(grace_period / 2)
     grace_period_to_kill_components = grace_period - grace_period_to_kill_gcs
 
-    # Kill evertyhing except GCS.
-    found, stopped, alive = kill_procs(
-        force, grace_period_to_kill_components, processes_to_kill[1:]
-    )
-    total_procs_found += found
-    total_procs_stopped += stopped
-    procs_not_gracefully_killed.extend(alive)
+    # Kill evertyhing except GCS, one kind at a time.
+    for process in processes_to_kill[1:]:
```

Comment:
If the kill order matters, could you add a comment to `RAY_PROCESSES` saying that the order matters and people shouldn't change it accidentally.

Reply:
done.